composer/utils/batch_helpers.py

```diff
@@ -53,8 +53,8 @@ def batch_get(batch: Any, key: Union[str, int, Callable, Any]):
             return itemgetter(*key)(batch)
         except TypeError:
             try:
-                return attrgetter(key)(batch)
-            except:
+                return attrgetter(cast(str, key))(batch)
+            except TypeError:
```

Comment:
I assume this is intentional but just thrown into PR

Reply:
Yeah, this one came along for the ride haha. VSCode started to complain about this.